src/pages/callouts/[id]/map.vue

```diff
@@ -409,6 +421,8 @@ onBeforeMount(async () => {
   responses.value = (await fetchResponsesForMap(props.id)).items.filter(
     (r): r is GetCalloutResponseMapDataWithAddress => !!r.address
   );
+
+  introOpen.value = true;
```

Comment:
Could you not just initialise `introOpen` with `true` then you don't need to do this?